misc/config/config.nydus.yaml

```diff
@@ -31,6 +31,8 @@ provider:
   gcpolicy:
       # size threshold that triggers GC, the oldest used blobs will be reclaimed if exceeds the size.
       threshold: 1000MB
+  # remote cache size for converted layers, default is 200.
```

Comment:
remote cache record capacity of converted layers, default is 200.

Reply:
fixed